src/ui/public/notify/banners/BANNERS.md

```diff
@@ -0,0 +1,148 @@
+# Banners
```

Comment:
These docs are wonderful. Can we add a link to them from https://github.com/pickypg/kibana/blob/feature/add-top-banners/src/ui/public/UI_SYSTEMS.md ?